app/src/main/java/de/tum/in/tumcampusapp/activities/wizard/WizNavStartActivity.java

```diff
@@ -196,6 +197,72 @@ public void onClick(DialogInterface dialog, int which) {
         }
     }
 
+    private void setDefaultCampus ( String faculty_number ) {
+
+        String Campus = "0";
+        switch (faculty_number)
+        {
```

Comment:
Could you please run AndroidStudio's auto-formatter ("Code -> Reformat Code") for this?

Reply:
Ok done, but almost nothing changed – https://github.com/foreverigor/TumCampusApp/commit/0c805282d31b5a02ab588b046149bbc452418da8

The project is built with -Werror so that's why the switch statement looks so clumsy (case fall-through is not allowed)

Reply:
Fallthrough is certainly allowed. I don't think -Werror / javas linting even detects this.
I think we already have a couple of fallthroughs in our code, e.g. see the [CardsAdapter](https://github.com/TCA-Team/TumCampusApp/blob/8219d041c7e5f525f74ec32db5d644b0e3c26b19/app/src/main/java/de/tum/in/tumcampusapp/adapters/CardsAdapter.java#L45)

Reply:
Dann schau ich mir das nochmals an, warum das bei mir nicht funktionierte.